war/src/test/java/io/lumeer/remote/rest/permissions/ProjectServicePermissionsIntegrationTest.java

```diff
@@ -0,0 +1,364 @@
+/*
+ * -----------------------------------------------------------------------\
+ * Lumeer
+ *
+ * Copyright (C) since 2016 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * -----------------------------------------------------------------------/
+ */
+
+package io.lumeer.remote.rest.permissions;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import io.lumeer.api.dto.JsonCollection;
+import io.lumeer.api.dto.JsonOrganization;
+import io.lumeer.api.dto.JsonPermission;
+import io.lumeer.api.dto.JsonProject;
+import io.lumeer.api.model.Collection;
+import io.lumeer.api.model.Permission;
+import io.lumeer.api.model.Project;
+import io.lumeer.api.model.Role;
+import io.lumeer.core.AuthenticatedUser;
+import io.lumeer.core.facade.OrganizationFacade;
+import io.lumeer.core.facade.ProjectFacade;
+import io.lumeer.engine.IntegrationTestBase;
+
+import org.jboss.arquillian.junit.Arquillian;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.List;
+import javax.inject.Inject;
+import javax.ws.rs.client.Client;
+import javax.ws.rs.client.ClientBuilder;
+import javax.ws.rs.client.Entity;
+import javax.ws.rs.core.GenericType;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+
+@RunWith(Arquillian.class)
+public class ProjectServicePermissionsIntegrationTest extends IntegrationTestBase {
```

Comment:
The same suggestion as in the organization tests applies here as well. You can inherit from `ServiceIntegrationTestBase` and simplify tests so you don't have to initialize and close the client.